compiler/src/iree/compiler/Pipelines/Pipelines.cpp

```diff
@@ -308,7 +312,15 @@ void buildIREEVMTransformPassPipeline(
         dispatchCreationOptions.enableAggressiveFusion;
     dispatchTransformOptions.enableFuseMultiUse =
         dispatchCreationOptions.enableFuseMultiUse;
+    // The pipeline option has higher priority.
     dispatchTransformOptions.dataTiling = dispatchCreationOptions.dataTiling;
+    if (pipelineOptions.dataTiling) {
+      dispatchTransformOptions.dataTiling = false;
+    }
```

Comment:
Also, feel free to ignore
```suggestion
dispatchTransformOptions.dataTiling = 
	dispatchCreationOptions.dataTiling && !dispatchCreationOptions.dataTiling;
```